addons/web/static/src/legacy/js/core/utils.js

```diff
@@ -403,7 +403,7 @@ const utils = {
      *  empty or if an error occurs.
      */
     getDataURLFromFile: function (file) {
-        if (!file) {
+        if (!file || !file.size) {
```

Comment:
Thanks for the fix.

Is it not sufficient for the reported use case?

Reply:
No, it is not sufficient as this it will give promise.reject which need to be handled during RPC call.